plaso/helpers/mac/darwin.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+"""Mac Darwin helper."""
```

Comment:
Please rename directory to macos to and use Mac instead of MacOS, to disambiguate with MAC address or MAC times